proto/celestia/qgb/v1/query.proto

```diff
@@ -8,11 +8,11 @@ import "gogoproto/gogo.proto";
 import "cosmos_proto/cosmos.proto";
 import "google/protobuf/any.proto";
 
-option go_package = "github.com/celestiaorg/celestia-app/x/qgb/types";
+option go_package = "github.com/celestiaorg/celestia-app/x/blobstream/types";
 
 // Query defines the gRPC querier service.
 service Query {
-  // Params queries the current parameters for the qgb module
+  // Params queries the current parameters for the blobstream module
```

Comment:
```suggestion
  // Params queries the current parameters for the Blobstream module
```

Reply:
I don't think we need to capitalize when referring to the state machine module